src/EditorFeatures/CSharpTest/CodeActions/GenerateFromMembers/GenerateConstructor/GenerateConstructorTests.cs

```diff
@@ -93,7 +93,7 @@ protected override object CreateCodeRefactoringProvider(Workspace workspace)
             await TestAsync(
 @"class Z { [|public int A { get ; private set ; } public string B { get ; private set ; }|] } ",
 @"class Z { public Z ( int a , string b ) { this . A = a ; this . B = b ; } public int A { get ; private set ; } public string B { get ; private set ; } } ",
-index: 0, options: new Dictionary<OptionKey, object> { { new OptionKey(SimplificationOptions.QualifyMemberAccessWithThisOrMe, "C#"), true } });
+index: 0, options: new Dictionary<OptionKey, object> { { new OptionKey(SimplificationOptions.QualifyPropertyAccess, LanguageNames.CSharp), true } });
```

Comment:
I think I said that in the last PR too. This is really cumbersome to have to do every time. Definitely a good place for a helper. Of course, if you add the helper, make sure to use it everywhere. :smile: